### DISCUSSION TITLE
[FEATURE REQUEST]:  make it work in vr

### DISCUSSION THREAD

**Discussion by xAstroBoy** [winstxnhdw/lc-hax#427] [Ideas]
### What happened?

Since this mod can be adjusted for vr, im thinking to make a small gui menu that can be held in hand and instead of using only commands to spawn enemies and such, some basic features can be ported over vr with a holdable gui prefab .

**Comment by winstxnhdw** [comment]
This is going to be tough to do without the Unity Editor.

**Comment by winstxnhdw** [comment]
Let's move this to discussions.
